src/A0_initialization.py

```diff
@@ -53,193 +101,199 @@ def create_parser():
                 FileNotFoundError(
-                    "Missing input json file! "
-                    "\n The input json file can not be found. Operation terminated."
+                    "Missing input json file!\n"
+                    "The input json file '{}' can not be found.\n"
+                    "Operation terminated.".format(JSON_FNAME)
                 )
             )
-
-    else:
-        if os.path.exists(path_input_file) is False:
+        json_files = [f for f in os.listdir(path_input_folder) if f.endswith(JSON_EXT)]
+        if len(json_files) > 1:
             raise (
-                NotADirectoryError(
-                    "Missing folder for inputs! "
-                    "\n The input folder can not be found. Operation terminated."
+                FileExistsError(
+                    "Two many json files in input folder ({})!\n"
+                    "Only the input json file '{}' should be present.\n"
+                    "Operation terminated.".format(", ".join(json_files), JSON_FNAME)
+                )
+            )
+    elif input_type == CSV_EXT:
+        path_input_file = os.path.join(path_input_folder, CSV_ELEMENTS, CSV_FNAME)
+        if os.path.exists(os.path.join(path_input_folder, CSV_ELEMENTS)) is False:
+            raise (
+                FileNotFoundError(
+                    "Missing folder for csv inputs! "
+                    "The input csv folder '{}' can not be found.\n"
+                    "Operation terminated.".format(CSV_ELEMENTS)
                 )
             )
 
-    return path_input_folder
+    return path_input_file
 
 
-def check_output_directory(path_output_folder, overwrite):
-    """
-    :param path_output_folder:
-    :param overwrite:
-    :return:
+def check_output_folder(path_input_folder, path_output_folder, overwrite):
+    """Enforces the rules for the output folder
+
+            An error is raised if the path_output_folder already exists, unless overwrite is set
+            to True. The path_output_folder is created if not existing and the content of
+            path_input_folder is copied in a folder named INPUTS_COPY.
+
+    :param path_input_folder: path to input folder
+    :param path_output_folder: path to output folder
+    :param overwrite: boolean indicating what to do if the output folder exists already
+    :return: the path to the folder stored in the output folder as copy of the input folder
     """
 
+    # make the path absolute
+    if REPO_PATH not in os.path.abspath(path_input_folder):
+        path_input_folder = os.path.join(REPO_PATH, path_input_folder)
     if REPO_PATH not in os.path.abspath(path_output_folder):
         path_output_folder = os.path.join(REPO_PATH, path_output_folder)
 
+    path_output_folder_inputs = os.path.join(path_output_folder, INPUTS_COPY)
+
     logging.debug("Checking for output folder")
     if os.path.exists(path_output_folder) is True:
         if overwrite is False:
-            user_reply = input(
```

Comment:
Sure we don't want this option anymore? I didn't follow conversations about this so excuse me if this is already solved.

I'd suggest one of these options:
1) keep user input for overwriting
2) Change error message: "Output folder exists and should not be overwritten. Please choose other folder." to "....... choose other folder or set `overwrite` to True.

Reply:
Yes it was agreed by @smartie2076 [here](https://github.com/rl-institut/mvs_eland/pull/138#discussion_r386444726)